process/suspension.html

```diff
@@ -58,9 +58,15 @@ <h4>Nearby</h4>
 </ul>
 <p>If the chair, team contact, CEO, or COO is the one accused of unacceptable behavior, they should not be involved in the decision to ban.</p>
 
-<p>2. Following violation of the <a href="/policies/code-of-conduct/">W3C Code of Conduct</a>, at least one warning, and at least one subsequent violation of the Code of Conduct, chairs, or team, may temporarily or permanently suspend participation by a person who has not formally joined the group.
+<p>2. Following violation of the <a href="/policies/code-of-conduct/">W3C Code of Conduct</a>, Chairs may gently remind about minor Code of Conduct violations, such as inadvertent microaggressions in public (meeting, email list) using language such as, "As a reminder, Person's pronoun's are they/them." This shows support for the microagressed person and demonstrates what will and won't be tolerated without pointing a finger at an individual. All other discussions of code of conduct violation should be in private unless there is an emergency (such as active assault in person). If a chair must remind a participant about the Code of Conduct:
```

Comment:
I understand the need to clarify how warnings are issued but this new paragraph removes the ability of a chair to suspend participation by a person who has not formally joined the group, such as limiting interaction on GitHub. Is that intended ?

Reply:
Thanks for pointing that out. I didn't mean to delete it. Edit coming.